Psychtoolbox/PsychHardware/NetStation.m

```diff
@@ -183,7 +213,53 @@
                 if n >= 100
                     warning('\nNetStationPC synchronization did not succeed within %.1f ms\nSynchronizatoin accuracy is %.1f ms\n',NSSynchLimit,df);
                 end
+                GETNTPSYNCED = 0;
+            end
+
+        %%% added by Andreas Widmann, widmann@uni-leipzig.de, 2017-07-19
```

Comment:
No need for those preambles and stuff at the end of the function - that's what git commit messages with author and e-mail are for.